en/publish/publish-baidugame.md

```diff
@@ -1,102 +1,100 @@
+- Automatic loading of remote resources, cache resources and cache resource versioning
 
-具体百度小游戏的申请入驻，开发准备，游戏提交，审核和发布流程可以参考 [百度小游戏注册指导文档](https://smartprogram.baidu.com/docs/game/)。
+For specific __Baidu Mini Game__ application, station development, game submission, review and release process can refer to [Baidu game registration guidance document](https://smartprogram.baidu.com/docs/game/).
 
-## 使用 Cocos Creator 发布百度小游戏
+## Using __Cocos Creator__ to release Baidu games
 
-### 准备工作
+### Prerequisites
 
-- 在 [百度开发者工具文档](https://smartprogram.baidu.com/docs/game/tutorials/howto/dev/) 里下载并安装百度开发者工具
-- 在手机的应用商店中下载并安装百度应用
-- 登录 [智能小程序平台](https://smartprogram.baidu.com/developer/index.html)，找到 appid
+- Download and install __Baidu Developer Tools__ in [Baidu Developer Tools Documentation](https://smartprogram.baidu.com/docs/game/tutorials/howto/dev/)
+- Download and install the __Baidu app__ in the app store of your phone
+- Log in to [smart applet platform](https://smartprogram.baidu.com/developer/index.html) and find appid
 
     ![](./publish-baidugame/appid.png)
 
-### 发布流程
+### Release process
 
-1. 在 **构建发布** 面板中选择 **发布平台** 为 **百度小游戏**，填入 **appid**，然后点击 **构建**。
+1. In the **Build Publishing** panel, select **Publishing Platform** for **Baidu Games**, fill in **appid**, then click **Build**.
 
     ![](./publish-baidugame/build.png)
-2. 构建完成后，会在发布包的目录下生成一个 **baidugame** 的百度小游戏工程文件夹，其中已经包含了百度小游戏环境的配置文件：`game.json` 和 `project.swan.json`
 
-    ![](./publish-baidugame/package.png)
-3. 使用 **百度开发者工具** 打开构建生成的 **baidugame** 文件夹，即可打开百度小游戏项目及预览调试游戏内容。**百度开发者工具** 的使用方式请参考 [百度开发者工具文档](https://smartprogram.baidu.com/docs/game/tutorials/howto/dev/)。
+2. After the build is completed, a __Baidu game__ project folder of **baidugame** will be generated in the directory of the release package, which already contains the configuration files of )__Baidu Mini Game__ environment: `game.json` and `project.swan.json`
 
-    ![](./publish-baidugame/preview.png)
+    ![](./publish-baidugame/package.png)
 
-**注意**：
+3. Use the **Baidu Developer Tools** to open the build **baidugame** folder to open the __Baidu game__ project and preview the game content. **How ​​to use Baidu Developer Tools** please refer to [Baidu Developer Tools Documentation](https://smartprogram.baidu.com/docs/game/tutorials/howto/dev/).
 
-- 请不要升级 **百度开发者工具** 到 **2.0.10** 版本。
-- 预览调试时若出现了 **当前版本的开发者工具无法发布小程序，请更新最新的开发者工具** 的提示，说明填写的 appid 是小程序的 appid，不是小游戏的 appid，请重新申请一个小游戏 appid。
+    ![](./publish-baidugame/preview.png)
 
-## 百度小游戏环境的资源管理
+  > __Note:__ Please do not upgrade **Baidu Developer Tools** to **2.0.10** version.
+  > __Note:__ **If the developer tool of the current version cannot be released during preview debugging, please update the prompt of the latest developer tool**, indicating that the appid is the appid of the applet, not the appid of the game, please re-apply for a small game appid.
```

Comment:
I don't understand what this really means. Can anyone help translate the Chinese better?

Reply:
Reference translation: When previewing debugging, if there is a prompt that **The current version of the developer tool can't publish mini program, please update the latest developer tools.**. Explain that the appid filled in the **Build** panel is the appid of the Baidu Mini Program, not the appid of the Baidu Mini Game, please re-apply for the appid of Baidu Mini Game.